pom.xml

```diff
@@ -51,8 +51,8 @@
         <jboss.snapshots.repo.id>jboss-snapshots-repository</jboss.snapshots.repo.id>
         <jboss.snapshots.repo.url>https://s01.oss.sonatype.org/content/repositories/snapshots/</jboss.snapshots.repo.url>
 
-        <quarkus.version>3.15.1</quarkus.version>
-        <quarkus.build.version>3.15.1</quarkus.build.version>
+        <quarkus.version>3.15.2</quarkus.version>
+        <quarkus.build.version>3.15.2</quarkus.build.version>
```

Comment:
Any reason these versions would ever be different? Perhaps we can de-duplicate this?

Reply:
AFAIK, it's mainly present due to productization tools and better manageability. There's a discussion somewhere around this with @ASzc.

Reply:
There are scenarios where the build-time version needs to be different than the runtime version. This is mostly when you're building Keycloak against a productized Quarkus, especially early builds thereof that might be incomplete with respect to the build-time-only stuff.

Splitting the properties makes this much easier to manage since Keycloak's build-only Quarkus dependencies are already identified